Essentials/src/com/earth2me/essentials/commands/EssentialsCommand.java

```diff
@@ -34,6 +34,7 @@
     protected transient IEssentials ess;
     protected transient IEssentialsModule module;
     protected static final Logger logger = Logger.getLogger("Essentials");
+    protected static List<UUID> confirmSetHome = new ArrayList<>();
```

Comment:
This should be a Set. 

EDIT: I made a comment under AgentTroll's comment that this should probably be added under the User (or something else)